tests/js/common/replication_api/shell-api-replication.js

```diff
@@ -0,0 +1,2524 @@
+      assertEqual(all["idleMaxWaitTime"], 3.5);
+
+      // refetch same data;
+      doc = arango.GET_RAW(cmd);
+
+      assertEqual(doc.code, 200);
+      all = doc.parsedBody;
+      assertEqual(all["endpoint"], "ssl://127.0.0.1:12345");
+      assertEqual(all["database"], "bar");
+      assertEqual(all["requestTimeout"], 12.5);
+      assertEqual(all["connectTimeout"], 26.3);
+      assertEqual(all["ignoreErrors"], 2);
+      assertEqual(all["maxConnectRetries"], 12);
+      assertEqual(all["sslProtocol"], 0);
+      assertEqual(all["chunkSize"], 1234567);
+      assertFalse(all["autoStart"]);
+      assertTrue(all["adaptivePolling"]);
+      assertFalse(all["autoResync"]);
+      assertFalse(all["includeSystem"]);
+      assertFalse(all["requireFromPresent"]);
+      assertFalse(all["verbose"]);
+      assertEqual(all["connectionRetryWaitTime"], 2.5);
+      assertEqual(all["initialSyncMaxWaitTime"], 4.3);
+      assertEqual(all["idleMinWaitTime"], 0.22);
+      assertEqual(all["idleMaxWaitTime"], 3.5);
+    },
+
+    ////////////////////////////////////////////////////////////////////////////////;
+    // state;
+    ////////////////////////////////////////////////////////////////////////////////;
+
+    test_checks_the_applier_state_again_2: function() {
+      // fetch state;
+      let cmd = api + "/applier-state";
+      let doc = arango.GET_RAW(cmd);
+
+      assertEqual(doc.code, 200);
+      let all = doc.parsedBody;
+      assertTrue(all.hasOwnProperty('state'));
+      assertTrue(all.hasOwnProperty('server'));
+
+      let state = all['state'];
+      assertFalse(state['running']);
+      assertTrue(state.hasOwnProperty("lastAppliedContinuousTick"));
+      assertTrue(state.hasOwnProperty("lastProcessedContinuousTick"));
+      assertTrue(state.hasOwnProperty("lastAvailableContinuousTick"));
+      assertTrue(state.hasOwnProperty("safeResumeTick"));
+
+      assertTrue(state.hasOwnProperty("progress"));
+      let progress = state['progress'];
+      assertTrue(progress.hasOwnProperty("time"));
+      assertMatch(/^([0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+Z)?$/, progress['time']);
+      assertTrue(progress.hasOwnProperty("failedConnects"));
+
+      assertTrue(state.hasOwnProperty("totalRequests"));
+      assertTrue(state.hasOwnProperty("totalFailedConnects"));
+      assertTrue(state.hasOwnProperty("totalEvents"));
+      assertTrue(state.hasOwnProperty("totalOperationsExcluded"));
+
+      assertTrue(state.hasOwnProperty("lastError"));
+      let lastError = state["lastError"];
+      assertTrue(lastError.hasOwnProperty("errorNum"));
+      assertTrue(state.hasOwnProperty("time"));
+      assertMatch(/^[0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+Z$/, state['time']);
+    },
+  };
+}
+
+////////////////////////////////////////////////////////////////////////////////;
+// logger;
+////////////////////////////////////////////////////////////////////////////////;
+function dealing_with_the_logger_Suite () {
```

Comment:
It looks like this suite is also already present here (https://github.com/arangodb/arangodb/pull/15935/files#diff-d30ab47493830791c04c990e1780dee036b030ecbe3fa694fc8903ccec338089R267 and following).
The difference between the 2 suites seems to be that they use different databases.

Reply:
we decided not to approach to de-duplicate the test.